kubernetes/worker-ubuntu16-04.yaml

```diff
@@ -38,8 +38,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-      - command: [/bb/tini, -v, -g, --, /bb/bb_runner, /config/runner-ubuntu16-04.jsonnet]
-        image: l.gcr.io/google/rbe-ubuntu16-04@sha256:b516a2d69537cb40a7c6a7d92d0008abb29fba8725243772bdaf2c83f1be2272
+      - command: [/bb/tini, -v, -g, --, /bb/bb_runner, /config/runner-ubuntu22-04.jsonnet]
+        image: ghcr.io/catthehacker/ubuntu:act-22.04@sha256:5f9c35c25db1d51a8ddaae5c0ba8d3c163c5e9a4a6cc97acd409ac7eae239448
```

Comment:
What's this container image? Who is catthehacker?

Reply:
That user is producing the images for https://github.com/nektos/act, used for running GitHub Actions locally. I've never used it before, but it sounds reasonable.

Reply:
Couldn't we use the official Ubuntu images here?

Reply:
It was about getting all the tooling, e.g. GCC. Maybe there are better variants, I am more than happy to change to something else.